### DISCUSSION TITLE
post/slack-angmondbot/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#93] [Announcements]
# post/slack-angmondbot/

슬랙봇 앙몬드를 통해 사내 기술 공유 문화를 활성화하고, 슬랙봇 개발 과정과 팁을 공유합니다.

https://tech.kakaopay.com/post/slack-angmondbot/

**Comment by hi-rachel** [comment]
너무 흥미롭게 잘 읽었습니다 👍 서버 개발 애기도 궁금하네요.
